src/main/java/com/google/api/gax/protobuf/PathTemplate.java

```diff
@@ -215,7 +215,19 @@ String separator() {
    * @throws ValidationException if there are errors while parsing the template.
    */
   public static PathTemplate create(String template) {
-    return new PathTemplate(parseTemplate(template));
+    return new PathTemplate(parseTemplate(template), true);
+  }
+
+  /**
+   * Creates a path template from a string. The string must satisfy the syntax
+   * of path templates of the API platform; see HttpRule's proto source.
+   *
+   * urlEncoding is used to control URL encoding of template variables.
+   *
+   * @throws ValidationException if there are errors while parsing the template.
+   */
+  public static PathTemplate create(String template, boolean urlEncoding) {
```

Comment:
I think I would prefer a differently-named create function instead of a boolean parameter.